### DISCUSSION TITLE
Add Docker installation

### DISCUSSION THREAD

**Discussion by AuthorShin** [spacedriveapp/spacedrive#178] [Feature Request]
Hello
When will you consider adding the docker installation method and then add image to hub.docker.com?
Can you please add/make docker-compose/docker CLI installation methods so that we can install spacedrive via docker management services like casaOS and Portainer?

**Comment by oscartbeaumont** [comment]
Currently, the docker image is only intended to be used internally by us (we are using it to deploy the backend used by the app embedded in the landing page).
When we are ready for people to deploy and use it, documentation will be provided and I will look into some of the ideas you have put forward and whether they make sense for us to maintain ourselves.

**Comment by kolaente** [comment]
Any updates on this?
